rebuildall.sh

```diff
@@ -307,10 +307,10 @@ if [ $just_gen_build_file -eq 1 ]; then
 fi
 
 
-if [ $no_check_code -eq 0 ]
-then
-  ./checkcodeformat.sh
-fi
+# if [ $no_check_code -eq 0 ]
```

Comment:
why this commented ?

Reply:
To simplify development iterations (or at least we thought so).

Restored.